### ISSUE TITLE
Problem install MagentoConnectorBundle

### ISSUE THREAD

**Issue by AlexanderHuyghebaert** [Agence-DnD/DnD-MagentoConnectorBundle#43]
Followed the tutorial and cleared the cache, but when I clear the cache, I get following error:
[Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException]  
  The service "dnd_magento_connector.reader.doctrine.product" has a dependency on a non-existent parameter "dnd_magento_connector.reader.doctrine.product.class". Did you mean one of these: "pim_base_connector.reader.doctrine.bulk_product.class", "pim_base_connector.rea  
  der.file.csv_product.class", "pim_base_connector.writer.doctrine.product.class", "pim_base_connector.reader.doctrine.product.class", "dnd_magento_connector.reader.orm.category.class", "dnd_magento_connector.reader.orm.attribute.class", "dnd_magento_connector.reader.o  
  rm.attribute_option.class", "dnd_magento_connector.writer.file.csv.class", "dnd_magento_connector.writer.file.csv_product.class"?

**Comment by florisgoumans** [comment]
Same issue here.

**Comment by AlexanderHuyghebaert** [comment]
Hi @florisgoumans, you can use https://github.com/akeneo-labs/EnhancedConnectorBundle instead.
This has replaced this Bundle

**Comment by DnD-Mimosa** [comment]
Hi

**Comment by DnD-Mimosa** [comment]
Hi @AlexanderHuyghebaert ,
Does this bundle export your images on your remote server ?
Hi @florisgoumans,
Do you use doctrine for your Akeneo PIM ? If you don't, that's why this class is not found. In this case, can you tell me what do you use ?
Regards,
Florian.

**Comment by florisgoumans** [comment]
@DnD-Mimosa Using doctrine/mongodb-odm for the product catalog, can that cause this issue?

Simply using a default Akeneo with MongoDB catalog.

**Comment by DnD-Mimosa** [comment]
@florisgoumans ,
Yes, that is why the class is not found. We have not already implemented the possibility to export your product with mongoDB.
Is it possible for you to pass your catalog on doctrine ?
Regards,
Florian.

**Comment by florisgoumans** [comment]
@DnD-Mimosa thats to bad, due to the size of the catalog (10+ locales, 30k products) mongodb is quite nescesary. Is this on your roadmap?

**Comment by DnD-Mimosa** [comment]
@florisgoumans ,

Is this on your roadmap?
Yes it is but it will not be implemented soon (around 1 month, you're free to make a PR if you want to do that functionnality if it's too long).

If you don't use Akeneo PIM for your images, you can use the Bundle recommended by @AlexanderHuyghebaert which support mongodb export. 

Warning : As I said, this bundle doesn't export your images on a remote server but PimGento need those images to be present in the media/import folder. 

Regards,
Florian.

**Comment by delyriand** [comment]
@DnD-Mimosa 
have you progress on the mongodb implementation on the DnD connector?

Regards,
Maxime.

**Comment by AlexanderHuyghebaert** [comment]
Hello,

Any progress on this,
Because we need export of images
